.github/workflows/tensorflow_submodule.yml

```diff
@@ -29,20 +29,20 @@ jobs:
       - name: GH CLI auth
         run: echo $GH_SERVICE_ACCOUNT_TOKEN | gh auth login --with-token
 
-      - name: Bump tensorflow, commit and push
+      - name: Bump tflite-micro, commit and push
         run: |
           export NEW_BRANCH=bot-$(date +%F_%H-%M)
           cd target_repo
           git checkout -b $NEW_BRANCH
-          pushd tensorflow
-          git checkout master
+          pushd tflite-micro
```

Comment:
will it leave the old directory in the tree? If so, please remember to remove it manually.

Reply:
oh, I see it's in another PR

Reply:
Actually, I addressed this in https://github.com/antmicro/tensorflow-zephyr-vexriscv-examples/pull/85